File: easy/FinalValueOfVariableAfterPerformingOperations/final_value_of_variable_after_performing_operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
 const finalValueAfterOperations = (operations) => {
  return operations.reduce((acc, item) => {
      const isInc = item.charAt(0) === '+' || item.charAt(2) === "+"
      return isInc ? ++acc : --acc
  }, 0)
};

File: easy/FinalValueOfVariableAfterPerformingOperations/final_value_of_variable_after_performing_operations.rs
impl Solution {
  pub fn final_value_after_operations(operations: Vec<String>) -> i32 {
      let mut sum = 0;
      for op in operations {
          let is_inc = op.chars().nth(0).unwrap() == '+' || op.chars().nth(2).unwrap() == '+';
          match is_inc {
              true => sum += 1,
              false => sum -=1
          }
      }
      sum
  }
}